core/src/main/resources/reference.conf

```diff
@@ -206,183 +77,188 @@ cassandra-journal {
-#     <settings...>
-#   }
-cassandra-snapshot-store {
-
-  # FQCN of the cassandra snapshot store plugin
-  class = "akka.persistence.cassandra.snapshot.CassandraSnapshotStore"
-
-  session-provider = "akka.cassandra.session.DefaultSessionProvider"
-
-  session-name = ""
-
-  write-profile = "cassandra-journal"
-  read-profile = "cassandra-journal"
-
-  # Name of the keyspace to be created/used by the snapshot store
-  keyspace = "akka_snapshot"
-
-  # Parameter indicating whether the snapshot keyspace should be auto created
-  # Not all Cassandra settings are configurable when using autocreate and for
-  # full control of the keyspace and table definitions you should create them
-  # manually (with a script).
-  keyspace-autocreate = true
+  write {
 
-  # Parameter indicating whether the snapshot tables should be auto created
-  # Not all Cassandra settings are configurable when using autocreate and for
-  # full control of the keyspace and table definitions you should create them
-  # manually (with a script).
-  tables-autocreate = true
+    # FQCN of the cassandra journal plugin
+    class = "akka.persistence.cassandra.journal.CassandraJournal"
 
-  # Name of the table to be created/used by the snapshot store.
-  # If the table doesn't exist it is automatically created.
-  table = "snapshots"
+    # Dispatcher for the plugin actor
+    plugin-dispatcher = "cassandra-plugin-default-dispatcher"
 
-  # Compaction strategy for the snapshot table
-  # Please refer to the tests for example configurations.
-  # Refer to http://docs.datastax.com/en/cql/3.1/cql/cql_reference/compactSubprop.html
-  # for more information regarding the properties.
-  table-compaction-strategy {
-    class = "SizeTieredCompactionStrategy"
-  }
+    # Name of the table to be created/used by the journal.
+    # If the table doesn't exist it is automatically created.
+    table = "messages"
 
-  # Name of the table to be created/used for journal config.
-  # If the table doesn't exist it is automatically created.
-  config-table = "config"
+    # Compaction strategy for the journal table.
+    # Please refer to the tests for example configurations.
+    # Refer to http://docs.datastax.com/en/cql/3.1/cql/cql_reference/compactSubprop.html
+    # for more information regarding the properties.
+    table-compaction-strategy {
+      class = "SizeTieredCompactionStrategy"
+    }
 
-  # Set this to on to only use Cassandra 2.x compatible features,
-  # i.e. if you are using a Cassandra 2.x server.
-  cassandra-2x-compat = off
+    # Name of the table to be created/used for storing metadata.
+    # If the table doesn't exist it is automatically created.
+    metadata-table = "metadata"
 
-  # Name of the table to be created/used for storing metadata.
-  # If the table doesn't exist it is automatically created.
-  metadata-table = "metadata"
+    # replication strategy to use. SimpleStrategy or NetworkTopologyStrategy
+    replication-strategy = "SimpleStrategy"
```

Comment:
I was thinking we could move all properties that are related to the automatic keyspace/table creation to a separate sub-section. Might make it more clear that they are only used once for that. wdyt?

Reply:
I think you already decided not to do this as we have diff keyspaces

Reply:
I skipped this idea, but added a comment to the properties that are related to auto-create.